src/components/ui-shell/_variables.scss

```diff
@@ -0,0 +1,4 @@
+// Use rem here to accomodate zoom behavior
+// Reference:
+// https://github.com/IBM/carbon-components/pull/1032#discussion_r210418579
+$unit: 0.5rem;
```

Comment:
Thank you for the change here! Do you want to use `rem()` mix-in in `_typography.scss`?

Reply:
Do you think it's necessary? I typically have only used that for sizes that I'd need a calculator to figure out the px value 😂 

Or is this more-so about the `units` function?